taskcluster/scripts/build/macosx_build.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+# Dependencies
+python3 scripts/tooltool.py --url http://taskcluster/tooltool.mozilla-releng.net fetch -m macos/qt-dependencies.tt.manifest
```

Comment:
Are you using this? It seems you pull qt from `https://github.com/mozilla-mobile/qt_static_macos` later? Or does it contain something different?